src/status_im2/common/floating_button_page/view.cljs

```diff
@@ -0,0 +1,113 @@
+  (let [insets (safe-area/get-insets)
+        height (reagent/atom 0)]
+    (reset! height (if show-keyboard? (if keyboard-shown keyboard-height 0) 0))
+    [rn/view {:style {:margin-top :auto}}
+     (cond
+       (and (> @height 0) show-background?)
+       [blur/view
+        (when keyboard-shown
+          {:blur-amount      34
+           :blur-type        :transparent
+           :overlay-color    :transparent
+           :background-color (if platform/android? colors/neutral-100 colors/neutral-80-opa-1-blur)
+           :style            (style/blur-button-container insets)})
+        children]
+
+       (and (> @height 0) (not show-background?))
+       [rn/view {:style (style/view-button-container true insets)}
+        children]
+
+       (not show-keyboard?)
+       [rn/view {:style (style/view-button-container false insets)}
+        children])]))
+
+(defn f-view
+  [{:keys [button-props button-label]} header children]
+  (reagent/with-let [show-keyboard?   (reagent/atom false)
+                     content-scroll-y (reagent/atom 0)
+                     show-listener    (oops/ocall rn/keyboard
+                                                  "addListener"
+                                                  (if platform/android?
+                                                    "keyboardDidShow"
+                                                    "keyboardWillShow")
+                                                  #(reset! show-keyboard? true))
+                     hide-listener    (oops/ocall rn/keyboard
+                                                  "addListener"
+                                                  (if platform/android?
+                                                    "keyboardDidHide"
+                                                    "keyboardWillHide")
+                                                  #(reset! show-keyboard? false))]
+
+    (let [{:keys [keyboard-shown
+                  keyboard-height]} (hooks/use-keyboard)
+          show-background?          (show-button-background keyboard-height
+                                                            keyboard-shown
+                                                            @content-scroll-y)]
+
+      [rn/view {:style style/page-container}
+       header
+       [rn/scroll-view
+        {:on-layout               (fn [event]
+                                    (let [height (oops/oget event "nativeEvent.layout.height")]
+                                      (reset! scroll-view-height height)
+                                      (reset! content-scroll-y 0)))
+         :on-scroll               (fn [event]
+                                    (let [y (oops/oget event "nativeEvent.contentOffset.y")]
+                                      (reset! content-scroll-y y)))
+         :scroll-event-throttle   64
+         :content-container-style {:flexGrow 1}}
```

Comment:
kebab-case for flex-grow

Reply:
for some reason this didn't work, I think it was something to do with the scroll view. at least that was my experience when implementing the create profile page